terraform/files/bin/flavor_disk.sh

```diff
@@ -31,6 +31,9 @@ usage()
 }
 
 if test -z "$1"; then usage; fi
+if ! command -v jq &>/dev/null; then
+  sudo apt update && sudo apt install -y jq
```

Comment:
Sidenote from @jschoone: Using apt-get might be safer (stable interface).

Reply:
`apt` is designed for interactive use and the "stable cli" is in reference to the possibility of changes breaking backwards compatibility if the "user friendly" aspect weighs stronger. `apt-get` is preferred for scripts to be safer in regards to future changes.

@jschoone i think the conclusion would be that while it should not be an issue in _this_ use-case, the usage of `apt-get` should be preferred to keep a clean base-line.